### DISCUSSION TITLE
Advanced Search for Files and Folders Using Events

### DISCUSSION THREAD

**Discussion by FelipeRearden** [Taitava/obsidian-shellcommands#180] [Show and tell]
Hello @Taitava !!!!!!!

I found one more interesting workflow to do with our amazing Events.

Using Obsidian URI we can create search applied to specific Folders and Files direct from the File Explorer or Quick Explorer 

### 1- Search inside a Specific Folder

`start “ ” ”obsidian://search?vault=MYVAULT&query=path:/{{event_folder_path:relative}}/ content:”`

Output: ignore
Event: Folder Menu

### 2- Search inside a Specific File 

`start “ ” ”obsidian://search?vault=MYVAULT&query=File:%22{{event_title}}%22 content:”`

Output: ignore
Event: File Menu

---

One more workflow to our collection :)

Have a great day !!!!

**Comment by Taitava** [comment]
Nice one! 😎👍 Thanks!
